### PR TITLE
General grammar and styling fixes

### PR DESCRIPTION
**Pull request by VOTProductions**
Consistency in code is important, and I've tried to make small changes in order to maintain it.

Please don't be too shouty at how terrible I am at Git, it's my first public commit.

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 import sys
 if not sys.version_info >= (3, 4):
-    print("openage requires Python 3.4 or higher")
+    print("Openage requires Python 3.4 or higher.")
```
Comment (TheJJ) [inline configure]: heh :) `openage` is actually meant to be a lowercase-word, so I'm not sure if we should capitalize it even on sentence beginnings.

File: libopenage/engine.cpp [hunk 2]
```diff
@@ -47,13 +47,13 @@ void Engine::create(util::Dir *data_dir, const char *windowtitle) {
...
 	} else {
-		throw Error{MSG(err) << "You tried to create another singleton engine instance!!111"};
+		throw Error{MSG(err) << "You tried to create another singleton engine instance!"};
 	}
 }
 
 void Engine::destroy() {
 	if (Engine::instance == nullptr) {
-		throw Error{MSG(err) << "You tried to destroy a nonexistant engine."};
+		throw Error{MSG(err) << "You tried to destroy a none-xistent engine."};
```
Comment (TheJJ) [inline libopenage/engine.cpp]: uh

File: configure (hunk 1 above)
Comment (VOTProductions) [inline configure]: Things like iPad are the exception, but you would capitalize Firefox and Steam. Stack Overflow is being inconsistent, as they are using "stackoverflow" for their logo, and yet their site is called "Stack Overflow". It's all over the place. :P

**Comment by TheJJ** [comment]
Thanks so far. I don't approve removing "funny" stuff, but otherwise the improvements look good. Can you now please [sqash those commits together](https://github.com/SFTtech/openage/blob/master/doc/contributing.md#rebasing---what-the-hell-is-it-and-why-do-i-need-it)?

**Comment by TheJJ** [comment]
It looks like you dropped one commit in the rebase, as the dash-typo is in there again.

**Comment by TheJJ** [comment]
Thanks. But before merging, you need to rebase in order to eliminate the merge commit and to fix the conflicts. You may also squash the commits into one.

Sorry I didn't notice earlier, but I'm not notified if somebody pushes new commits to a pull request.

File: configure (hunk 1 above)
Comment (franciscod) [inline configure]: remember to turn this back to lowercase :)

File: copying.md [hunk 3]
```diff
@@ -61,6 +61,7 @@ _the openage authors_ are:
 | Miguel Kasparick            | miguellissimo               | miguellissimo@gmail.com               |
 | Darren Strash               | darrenstrash                | darren.strash@gmail.com               |
 | Kyle Robbertze              | paddatrapper                | paddatrapper@gmail.com                |
+| Robert Chomicz              | Votato, vot                 | votproductions@gmail.com              |\
```
Comment (franciscod) [inline copying.md]: whoops, `\` at the end

**Comment by franciscod** [comment]
I'm not sure if I like bullet lists with periods:

- I don't like this.
- The bullet list already has items and don't need extra punctuation.
- You already see when one starts and another ends very clearly.

- So this is better!
- No periods
- Cleaner
- **you even save bytes**

**Comment by TheJJ** [comment]
Indeed, it looks better without the periods I'd say.

**Pull request closed without merging**